docs/getting-started/using-pubsub-subprotocol/python-work-with-subprotocols.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: docs
```

Comment:
also add to toc https://github.com/Azure/azure-webpubsub/blob/main/docs/_data/nav.yml#L17 and overview page https://github.com/Azure/azure-webpubsub/blob/main/Readme.md#step-by-step-tutorials and https://github.com/Azure/azure-webpubsub/blob/main/docs/getting-started/introduction.md#step-by-step-tutorials and https://github.com/Azure/azure-webpubsub/blob/main/docs/getting-started/quickstart.md#more-samples-to-try?